LiskHQ/lisk-mobile
# Use Node.js 8.13.0 - Closes #413

# What was the bug or feature?
Node.js `8.12.0` was being used while `8.13.0` is out with important security updates.

### How did I fix it?
Bumped version in `.nvmrc` file.

## Type of change
Please choose one of the options:
- Bug fix (a non-breaking change which fixes an issue)

### How to test it?
Check `.nvmrc` file and the Jenkins build to ensure Node.js version `8.13.0` is being used.

# Checklist:
- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- I have added tests that prove my fix is effective or that my feature works
- New and existing unit tests pass locally with my changes
